### PR TITLE
Update installation docs to describe miniconda install

### PR DESCRIPTION
**Pull request by jchodera**
This PR updates the installation docs to describe installation via `miniconda`.

### CODE REVIEW

**Comment by jchodera** [comment]
@andrrizzi: Can you review and merge?

File: docs/installation.rst [hunk 1]
```diff
@@ -7,17 +7,30 @@ Installing via `conda`
 ======================
 
 The simplest way to install YANK is via the `conda <http://www.continuum.io/blog/conda>`_  package manager.
-Packages are provided on the `omnia binstar channel <http://binstar.org/omnia>`_ for Linux, OS X, and Win platforms.
-The `yank binstar page <https://binstar.org/omnia/yank>`_ has useful instructions and download statistics.
+Packages are provided on the `omnia Anaconda Cloud channel <http://anaconda.org/omnia>`_ for Linux, OS X, and Win platforms.
+The `yank Anaconda Cloud page <https://anaconda.org/omnia/yank>`_ has useful instructions and `download statistics <https://anaconda.org/omnia/yank/files>`_`.
```
Comment (andrrizzi) [inline docs/installation.rst]: The last link on this line is broken. Just an issue with the \` characters.

File: docs/installation.rst [hunk 2]
```diff
@@ -68,21 +81,45 @@ Testing your installation
...
+Optional Tools
+--------------
+
+The `OpenEye toolkit and Python wrappers <http://www.eyesopen.com/toolkits>`_ can be installed to enable free energy calculations to be set up directly from `multiple supported small molecule formats <https://docs.eyesopen.com/toolkits/python/oechemtk/molreadwrite.html#file-formats>`_, including
+
+* Tripos mol2
+* SDF
+* SMILES
+* IUPAC names
+* PDB
```
Comment (andrrizzi) [inline docs/installation.rst]: `PDB` and `mol2` support does not depend on OpenEye if one doesn't want to use the OE charge scheme.

File: docs/installation.rst [hunk 3]
```diff
@@ -105,22 +142,35 @@ Required
...
-
-* `mpi4py <http://mpi4py.scipy.org/>`_ is needed if  MPI support is desired.
+* `mpi4py <http://mpi4py.scipy.org/>`_ is needed if `MPI support <https://de.wikipedia.org/wiki/Message_Passing_Interface>`_ is desired.
 
-.. note:: The ``mpi4py`` installation must be compiled against the system-installed MPI implementation used to launch jobs.
+.. note:: The ``mpi4py`` installation must be compiled against the system-installed MPI implementation used to launch jobs. Using the ``conda`` version of ``mpi4py`` together with the ``conda``-provided ``mpirun`` is the simplest way to avoid any issues.
 
-* The `OpenEye toolkit and Python wrappers <http://www.eyesopen.com/toolkits>`_ can be used to enable free energy calculations to be set up directly from any supported OpenEye format, including mol2, PDB, ChemDraw, and many more (requires academic or commercial license).
+* The `OpenEye toolkit and Python wrappers <http://www.eyesopen.com/toolkits>`_ can be used to enable free energy calculations to be set up directly from multiple supported OpenEye formats, including Tripos mol2, PDB, SMILES, and IUPAC names (requires academic or commercial license).
```
Comment (andrrizzi) [inline docs/installation.rst]: Same here (`mol2` and `PDB` do not require OE)

File: docs/quickstart.rst [hunk 4]
```diff
@@ -5,36 +5,29 @@ Quickstart (for the impatient)
...
 
-.. code-block:: none
+.. code-block:: bash
 
-   $ yank prepare binding amber --setupdir=setup --ligand="resname BEN" --store=output --iterations=1000 \
-     --restraints=harmonic --gbsa=OBC2 --temperature=300*kelvin --verbose
+   $ yank script --yaml=yank.yaml
 
-Run the simulation in serial mode with:
+Alternatively, the simulation can be `run in MPI mode <running-yank>`_ if you have multiple GPUs available.
```
Comment (andrrizzi) [inline docs/quickstart.rst]: Should the link here be only `<running>`?

**Comment by andrrizzi** [comment]
Just made inline comments. The rest looks awesome!

File: docs/installation.rst (hunk 1 above)
Comment (jchodera) [inline docs/installation.rst]: Fixed!

File: docs/installation.rst (hunk 2 above)
Comment (jchodera) [inline docs/installation.rst]: Corrected!

File: docs/installation.rst (hunk 3 above)
Comment (jchodera) [inline docs/installation.rst]: Fixed.

File: docs/quickstart.rst (hunk 4 above)
Comment (jchodera) [inline docs/quickstart.rst]: Fixed

**Comment by jchodera** [comment]
Thanks for the comments! Should be all fixed now!

**Comment by andrrizzi** [comment]
Looks great! Merging.

**Comment by jchodera** [comment]
Weird. The docs were not updated at http://getyank.org. Trying to track down why.

**Comment by jchodera** [comment]
Looks like I had disabled pushing the docs to S3. Not sure how that happened. Fixing in a separate PR.

**Pull request merged**